### DISCUSSION TITLE
Translation of Admin Panel possible in Beta?

### DISCUSSION THREAD

**Discussion by davideperozzi** [payloadcms/payload#7142] [Q&A] [answered]
I was testing the Beta (.63) for a smaller project and wanted the backend to be translated to German. But right now, I don't see a language selector in the panel or a way to change the language in the backend in general (localization for content works!). 

![image](https://github.com/user-attachments/assets/122d7266-25f8-450a-b5ea-98191c0860ce)

I assume it's still WIP? Couldn't find any information about it.
After I've added the localization, there's a language switcher (for the content I assume?). 
But it has no effect on the admin UI as you can see here:

![image](https://github.com/user-attachments/assets/70f24b78-32fb-4787-89b2-fa2467b4e532)

Maybe somebody noticed the same with the beta or maybe I'm doing something wrong?

**Comment by jmikrut** [comment, marked as answer]
Hey @davideperozzi —

You need to pass all the languages you'd like to support into Payload.

Take a look at the docs here:
https://payloadcms.com/docs/beta/configuration/i18n#adding-languages

How's that?

**Comment by davideperozzi** [comment]
@jmikrut Awesome, thank you - it works. I was checking the latest docs, didn't know there was an option there to switch to beta. Can't wait for the stable release!
